server/src/main/java/org/cloudfoundry/identity/uaa/zone/JdbcIdentityZoneProvisioning.java

```diff
@@ -117,6 +143,21 @@ public IdentityZone create(final IdentityZone identityZone) {
         return retrieveIgnoreActiveFlag(identityZone.getId());
     }
 
+    @Override
+    public OrchestratorZoneEntity createOrchestratorZone(String identityZoneId, String orchestratorZoneName) {
+        try {
+            jdbcTemplate.update(CREATE_ORCHESTRATOR_ZONE_SQL, ps -> {
```

Comment:
Method jdbcTemplate.update(...), is this creating a new record or updating existing one?

Reply:
JdbcTemplate update method will do create/update based on the query we are passing, if we pass the insert query it will create record, if we pass the update query it will update the record.